pkg/util/util.go

```diff
@@ -124,7 +133,17 @@ func ConvertStringToInt32(s string) int32 {
 // IsSSLEnabledForInternalCommunication checks if ssl is enabled for internal communication
 func IsSSLEnabledForInternalCommunication(l []v1beta1.InternalListenerConfig) (enabled bool) {
 	for _, listener := range l {
-		if listener.Type.IsSSL() {
+		if listener.UsedForInnerBrokerCommunication && listener.Type.IsSSL() {
+			enabled = true
+			break
+		}
+	}
+	return enabled
+}
+
+func IsCruiseControlMetricsNeedSSL(l []v1beta1.InternalListenerConfig, brokerPort int32) (enabled bool) {
```

Comment:
We should rename this function since this not relates to CC metrics at all as far as I can tell from the code.

Reply:
True, but its only used when we check that cruise control metrics need SSL or not so somehow relates to it. What do you think should we rename this for sure